projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -149,6 +152,10 @@ function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, cl
 				classNames={ formClassnames }
 			/>
 		);
+
+		// Remove padding from the empty form.
+		blockProps.style.paddingTop = 0;
+		blockProps.style.paddingBottom = 0;
```

Comment:
What about left and right padding? If you are resetting padding for the placeholder, shouldn't you reset it all?

Reply:
No but the user might apply some and these changes forcibly reset that user selection. So either you override the user's choices completely or you don't.